### ISSUE TITLE
stage-deadline errors for RFDK 0.18.0

### ISSUE THREAD

**Issue by aws-painec** [aws/aws-rfdk#177]
# Description

There are two issues with the `stage-deadline` script in the RFDK 0.18.0 release.

## 1. Failed import of `aws-sdk`

Running the `stage-deadline` script in isolation is useful for quickly importing Deadline assets for use by the app, and also is used in our integration tests, and is also currently the expected way to stage Deadline for a Python app. This does not work for RFDK 0.18.0 and it outputs the error:

    Cannot find module 'aws-sdk'

### Reproduction Steps

1. Clone the RFDK
1. `cd` to the `examples/deadline/All-In-AWS-Infrastructure-Basic/python` directory
1. Install dependencies with `python setup.py install`
1. Execute `npx --package=aws-rfdk@0.18.0 stage-deadline 10.1.9.2`

### Actual Result

```
internal/modules/cjs/loader.js:968
  throw err;
  ^

Error: Cannot find module 'aws-sdk'
Require stack:
- /local/home/painec/.npm/_npx/18416/lib/node_modules/aws-rfdk/lib/core/lambdas/nodejs/lib/dynamodb/composite-table.js
- /local/home/painec/.npm/_npx/18416/lib/node_modules/aws-rfdk/lib/core/lambdas/nodejs/lib/dynamodb/index.js
- /local/home/painec/.npm/_npx/18416/lib/node_modules/aws-rfdk/lib/core/lambdas/nodejs/lib/custom-resource/dynamo-backed-resource.js
- /local/home/painec/.npm/_npx/18416/lib/node_modules/aws-rfdk/lib/core/lambdas/nodejs/lib/custom-resource/index.js
- /local/home/painec/.npm/_npx/18416/lib/node_modules/aws-rfdk/lib/core/lambdas/nodejs/version-provider/handler.js
- /local/home/painec/.npm/_npx/18416/lib/node_modules/aws-rfdk/lib/core/lambdas/nodejs/version-provider/index.js
- /local/home/painec/.npm/_npx/18416/lib/node_modules/aws-rfdk/bin/stage-deadline.js
- /local/home/painec/.npm/_npx/18416/lib/node_modules/aws-rfdk/bin/stage-deadline
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:965:15)
    at Function.Module._load (internal/modules/cjs/loader.js:841:27)
    at Module.require (internal/modules/cjs/loader.js:1025:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (/local/home/painec/.npm/_npx/18416/lib/node_modules/aws-rfdk/lib/core/lambdas/nodejs/lib/dynamodb/composite-table.js:10:19)
    at Module._compile (internal/modules/cjs/loader.js:1137:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)
    at Module.load (internal/modules/cjs/loader.js:985:32)
    at Function.Module._load (internal/modules/cjs/loader.js:878:14)
    at Module.require (internal/modules/cjs/loader.js:1025:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/local/home/painec/.npm/_npx/18416/lib/node_modules/aws-rfdk/lib/core/lambdas/nodejs/lib/dynamodb/composite-table.js',
    '/local/home/painec/.npm/_npx/18416/lib/node_modules/aws-rfdk/lib/core/lambdas/nodejs/lib/dynamodb/index.js',
    '/local/home/painec/.npm/_npx/18416/lib/node_modules/aws-rfdk/lib/core/lambdas/nodejs/lib/custom-resource/dynamo-backed-resource.js',
    '/local/home/painec/.npm/_npx/18416/lib/node_modules/aws-rfdk/lib/core/lambdas/nodejs/lib/custom-resource/index.js',
    '/local/home/painec/.npm/_npx/18416/lib/node_modules/aws-rfdk/lib/core/lambdas/nodejs/version-provider/handler.js',
    '/local/home/painec/.npm/_npx/18416/lib/node_modules/aws-rfdk/lib/core/lambdas/nodejs/version-provider/index.js',
    '/local/home/painec/.npm/_npx/18416/lib/node_modules/aws-rfdk/bin/stage-deadline.js',
    '/local/home/painec/.npm/_npx/18416/lib/node_modules/aws-rfdk/bin/stage-deadline'
  ]
}
```
### Expected Result

The script executes and downloads Deadline assets to the `stage` folder

### Workaround
Typescript apps can locally run `npx stage-deadline 10.1.9.2`, but Python apps cannot, so for Python the user has to stage Deadline elsewhere and copy the staging folder over

### Environment

  - **RFDK Version:** 0.18.0
  - **Deadline Version:** 10.1.9.2
  - **Node.js Version:** 12.18.3
  - **OS               :** AL2

### Other

I tried installing the missing module with `npm install -g aws-sdk` but it didn't help.

## 2. HTTP 301 Redirect from HTTP -> HTTPS causes abort

When the `aws-sdk` is available in the node search path, a different error message is displayed:

    Expected status code 200, but got 301

### Reproduction Steps

1.  Clone the RFDK
1.  Build the project (`./build.sh`)
1.  `cd examples/deadline/All-In-AWS-Infrastructure-Basic/ts`
1.  Run `npx stage-deadline`

### Actual Result

```
Expected status code 200, but got 301
```

### Expected Result

The script executes and downloads Deadline assets to the `stage` folder

--- 

This is :bug: Bug Report

**Comment by jusiskin** [comment]
I've conducted an investigation and got past the:

```
Cannot find module 'aws-sdk'
```

error message. Once past that, there is a second error:

```
Expected status code 200, but got 301
```

I will be using this same issue to track that error as well and will modify the description/title

**Comment by jusiskin** [comment]
This is now resolved with the release of 0.18.1. I've tested using:

```bash
npx --package=aws-rfdk@0.18.1 stage-deadline 10.1.9.2
```

and confirmed that the command now works.
